source/_integrations/input_boolean.markdown

```diff
@@ -12,7 +12,7 @@ ha_domain: input_boolean
 
 The `input_boolean` integration allows the user to define boolean values that can be controlled via the frontend and can be used within conditions of automation. This can for example be used to disable or enable certain automations.
 
-The preferred way to configure input booleans is via the user interface at **Configuration** -> **Helpers**. Click the add button and then choose the "**Toggle**" option.
+The preferred way to configure input booleans is via the user interface at **Configuration** -> **Helpers**. Click the add button and then choose the "**Toggle**" option. (The "**Toggle**" option might be grayed out if you don't have an `input_boolean:` section already in your `configuration.yaml` file or an `!include` for input_boolean.)
```

Comment:
Thanks, @beatified!

However, this is not entirely correct because of two things.

1) The `input_boolean` integration is part of the `default_config` configuration already. Unless one throws that out manually, one would end up having such problem.
2) It doesn't matter if one uses `!include` (or any other include list type directive) or just write a list directly under the `input_boolean` item itself.